cpp/test/cpu_operations_test/logical_or_test.cc

```diff
@@ -30,8 +30,39 @@
 
 Nice::Matrix<bool> m1;  // First boolean matrix
 Nice::Matrix<bool> m2;  // Second boolean matrix
+Nice::Matrix<bool> m3;  // Stores LogicalOr return value for matrices
+Nice::Vector<bool> v1;  // First boolean vector
+Nice::Vector<bool> v2;  // Second boolean vector
+Nice::Vector<bool> v3;  // Stores LogicalOr return value for vectors
 
+// Tests the basic functionality of LogicalOr for Matrices
 TEST(MyTest, LogicalOrMatrixFunctionality) {
   m1.setRandom(4,4);
   m2.setRandom(4,4);
+  m3 = Nice::CpuOperations<bool>::LogicalOr(m1, m2);
+  ASSERT_TRUE( m3.isApprox( ( m1.array() || m2.array() ).matrix() ) );
+}
+
+// Tests to see if LogicalOr for matrices throws an error when matrices are of
+// different sizes
+TEST(MyTest, LogicalOrMatrixDiffSize) {
```

Comment:
The first argument for TEST is expected to be the test case name. So instead of "MyTest", could you rename it to something like "LogicalOrTest" or "LogicalAndTest"? This would make the output message of "make test ARGS="-V"" more intuitive.